### PR TITLE
Explicitly stop/start docker to fix local-up-cluster.sh

### PR DESCRIPTION
**Pull request by dims**
Fix for https://github.com/kubernetes/kubernetes/issues/122183

#### What type of PR is this?

/kind bug

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by saschagrunert** [comment]
Looks like we have a networking issue in the test setup :thinking:

**Comment by pacoxu** [comment]
> Looks like we have a networking issue in the test setup 🤔

That seems to be unrelated with this change. It means that this PR only fix the `pull-kubernetes-local-e2e` CI partially. 

LGTM

**Comment by saschagrunert** [comment]
> > Looks like we have a networking issue in the test setup 🤔
> 
> @saschagrunert That seems to be unrelated with this change. It means that this PR only fix the `pull-kubernetes-local-e2e` CI partially.
> 
> LGTM

Yeah I know, LGTM from my side as well. Brings us closer to a solution :+1:

**Review by saschagrunert — APPROVED** [review]

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -x
```
Comment (pohly) [inline hack/local-up-cluster.sh:17]: @dims: this was probably added for debugging and shouldn't have been committed?

Comment (HirazawaUi) [reply hack/local-up-cluster.sh:17]: Also have doubts about it, `set -x` outputs too much information.

Comment (dims) [reply hack/local-up-cluster.sh:17]: still need it. we are debugging the `docker stop` (we added a timeout)

**Pull request merged**